Library/Homebrew/cask/cmd/home.rb

```diff
@@ -4,6 +4,8 @@ module Cask
   class Cmd
     class Home < AbstractCommand
       def run
+        odeprecated "brew cask home", "brew home"
```

Comment:
Leave this commented out for now (but the code as-is) and it'll mean I can merge the PR now but will only deprecate it when I do all the deprecations for 2.5.